scipy/interpolate/_interpolate.py

```diff
@@ -2360,7 +2381,8 @@ class RegularGridInterpolator:
     Parameters
     ----------
     points : tuple of ndarray of float, with shapes (m1, ), ..., (mn, )
-        The points defining the regular grid in n dimensions.
+        The points defining the regular grid in n dimensions. The points in
+        all dimensions must be strictly ascending or descending.
```

Comment:
It took me a while to parse this statement, esp _all dimensions_. So the idea is that for each `j \in [0, ndim)`, `points[j]` is monotone, so that either `points[j][k] > points[j][k+1]` for all `k` or  `points[j][k] < points[j][k+1]` ?
Then maybe say _the points in each dimension (i.e. every elements of the `points` tuple) must be_?